code/modules/food_and_drinks/kitchen_machinery/coffeemaker.dm

```diff
@@ -0,0 +1,693 @@
+#define BEAN_CAPACITY 10 //amount of coffee beans that can fit inside the impressa coffeemaker
+
+/obj/machinery/coffeemaker
```

Comment:
What's the point of the coffee maker that uses cartridges if a better more reasonable one exists that uses the coffee beans?

Reply:
So this is definitely an interesting question, first and foremost I like to see it as a preference- some people like cheap and easy coffee that you can pop a cartridge into and pour out without the need for beans. They come in their own 'unique' flavours as well for snobs, whereas the impressa machine which uses beans will be regular, or robusta unless more are added. Mostly just a matter of style that I don't see much point in not having both available.

Secondly, the first machine came out, well, first and is the baseline for the impressa and removing it would be really annoying and im a bit lazy, though if push comes to shove it can be done.

Reply:
i think its fine to have both the cups and regular beans personally

Reply:
It's just an odd choice. Not cowbot's choice, but an odd one regardless

Reply:
What would you like me to do in this situation?

Reply:
Don't worry about changing it